internal/utils/yaml_utils.go

```diff
@@ -0,0 +1,24 @@
+package utils
+
+import (
+	"regexp"
+	"strings"
+)
+
+func ReplaceYAMLField(content, oldField, newField string) string {
```

Comment:
@Arpan-206 let's add tests for these and also ensure that they handle a variety of cases (like with single quotes, double quotes etc.)